### DISCUSSION TITLE
The internet is slipping out of our reach

### DISCUSSION THREAD

**Discussion by giscus** [srijan-paul/srijan-paul.github.io#5] [Announcements]
# The internet is slipping out of our reach

With AI encroaching every open forum on the internet, the future of human discourse is bleak.

https://injuly.in/blog/darker-internet/

**Comment by tanmay1193** [comment]
Really nice blog post. Loved reading it. And yes first 3 links I always see are ads. everytime I look for review I add reddit at the end :)

**Comment by brawll** [comment]
so true man, I have been personally using Crossquery extension by Paraschorpra and this we how we browse web I guess, almost on the verge of purchasing a Kagi subscription.
